platform/windows/os_windows.cpp

```diff
@@ -1217,6 +1217,53 @@ Error OS_Windows::shell_open(String p_uri) {
 	}
 }
 
+Error OS_Windows::shell_show_in_explorer(String p_path, bool p_open_folder) {
+	if (p_path.begins_with("file://")) {
+		p_path = p_path.trim_suffix("file://");
+	}
+
+	bool open_folder = false;
+	if (DirAccess::dir_exists_absolute(p_path) && p_open_folder) {
+		open_folder = true;
+	}
+
+	if (p_path.begins_with("\"")) {
+		p_path = String("\"") + p_path;
+	}
+	if (p_path.ends_with("\"")) {
+		p_path = p_path + String("\"");
+	}
+	p_path = p_path.replace("/", "\\");
+
+	INT_PTR ret = OK;
+	if (open_folder) {
+		ret = (INT_PTR)ShellExecuteW(nullptr, nullptr, LPCWSTR(String("explorer.exe").utf16().get_data()), LPCWSTR(p_path.utf16().get_data()), nullptr, SW_SHOWNORMAL);
```

Comment:
```suggestion
		ret = (INT_PTR)ShellExecuteW(nullptr, nullptr, LPCWSTR("explorer.exe"), LPCWSTR(p_path.utf16().get_data()), nullptr, SW_SHOWNORMAL);
```
Wouldn't this work? What's the point of constructing a String and then deconstructing it?

Reply:
```suggestion
		ret = (INT_PTR)ShellExecuteW(nullptr, nullptr, L"explorer.exe", LPCWSTR(p_path.utf16().get_data()), nullptr, SW_SHOWNORMAL);
```